### PR TITLE
chore: #1362 update README.me and documenting what breaking changes are

### PR DESCRIPTION
**Pull request by droshev**
fixes #1362

### CODE REVIEW

**Comment by joseegm** [comment]
Test Version for this PR was deployed

Built with commit f42e8b7915055f72414a6adc6aeaa7269a947e6f

https://deploy-preview-1362--fundamental.netlify.com

File: README.md [hunk 1]
```diff
@@ -93,6 +93,17 @@ The `fiori-fundamentals` library follows [Semantic Versioning](https://semver.or
...
 
+The following circumstances will be considered a MAJOR or BREAKING change:
+* Droppping existing classnames, css variables, color names, color groups, spacing parameters
+* The existing underlying HTML markup of a component is altered
+* An existing unit test is altered to account for either of the above
+
+The following circumstances will NOT be considered a MAJOR or BREAKING change:
+* Introducing new  classnames, css variables, color names, color groups, spacing parameters
+* Non-visual HTML attribute changes/additions (such as `role`, `aria-*`, `data-*`)
+* An existing unit test is altered to account for non-visual HTML attribute changes/additions (such as `role`, `aria-*`, `data-*`)
```
Comment (bcullman) [inline README.md]: (nit) I'd like this bullet point to be the final one.

File: README.md [hunk 2]
```diff
@@ -93,6 +93,17 @@ The `fiori-fundamentals` library follows [Semantic Versioning](https://semver.or
...
+* Modifying CSS properties and values of existing classnames.
```
Comment (bcullman) [inline README.md]: "Adding or modifying CSS properties"...

Comment (droshev) [reply README.md]: updated

File: README.md (hunk 1 above)
Comment (droshev) [reply README.md]: done

**Pull request merged**